corehq/apps/domain/management/commands/unlink_project_spaces.py

```diff
@@ -0,0 +1,84 @@
+from __future__ import absolute_import
+from __future__ import unicode_literals
+from django.core.management.base import BaseCommand
+
+from corehq.apps.domain.models import Domain
+from corehq.apps.linked_domain.models import DomainLink
+from corehq.apps.app_manager.models import Application
+from six.moves import input
+
+
+class Command(BaseCommand):
+    help = "Unlinks linked project spaces and converts the downstream app into a standalone app."
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            'master_domain_name',
+            help='The name of the master project space'
+        )
+        # parser.add_argument(
+        #     'master_app_name',
+        #     help='The name of the master app'
+        # )
+        parser.add_argument(
+            'linked_domain_name',
+            help='The name of the linked project space'
+        )
+        parser.add_argument(
+            'linked_app_name',
+            help='the name of the linked app'
+        )
+
+    # def handle(self, master_domain_name, master_app_name, linked_domain_name, linked_app_name, **options):
+    def handle(self, master_domain_name, linked_domain_name, linked_app_name, **options):
+        master_domain = Domain.get_by_name(master_domain_name)
+        linked_domain = Domain.get_by_name(linked_domain_name)
+        if not master_domain:
+            print('domain with name "{}" not found'.format(master_domain_name))
+            return
+        if not linked_domain:
+            print('domain with name "{}" not found'.format(linked_domain_name))
+            return
+
+        try:
+            domain_link = DomainLink.objects.get(
+                linked_domain=linked_domain_name, master_domain=master_domain_name
+            )
+        except DomainLink.DoesNotExist:
+            print('No DomainLink found for master project space {} and linked project space {}.'.format(
+                master_domain_name, linked_domain_name
+            ))
+            return
+
+        linked_apps = []
+        for app in linked_domain.applications():
```

Comment:
As mentioned above, I think you should pull out a specific application with an ID, which means this whole block can be removed